http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!-- If you modify CLUSTER table, please make sure to notify @dspfieldeng on slack if there's updates to any of the fields in https://github.com/broadinstitute/workbench-analytics/blob/master/dbconfig.py#L136-L154 -->
-<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
I usually do not touch these lines, was there a reason to change them?

Reply:
I think the formatter just took over at this point when I saved the file and wrapped them onto newlines.  I thought it looked a little bit easier to read so I didn't fight it.

Reply:
I put this back in d31af9527337378aeecdcb19bc44a3685143de3f to avoid potentially having dueling workspaces/IDEs.